docs/UserGuide.md

```diff
@@ -250,25 +278,30 @@ Format: `exit`
 
 ### Saving the data
 
-Productiv data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+*Productiv* data are saved in your computer's hard disk automatically after any command that changes it. 
+There is no need to save manually.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it 
-creates with the file that contains the data of your previous Productiv home folder.
+creates with the file that contains the data of your previous *Productiv* app.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
 Action | Format, Examples
 --------|------------------
-**Mode** | `mode`
-**Add (Contact)** | `add ROLE n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [t/TAG]` <br> e.g., `add stk n/Jenny Ho p/98981212 e/jennyho@pmail.com`
-**Edit (Meeting)** | `edit INDEX [t/TITLE] [desc/DESCRIPTION] [from/FROM] [to/TO] [c/CONTACTS] [l/LOCATION]` <br> e.g., `edit 2 l/Blk C Level 8 Meeting Room 1`
-**Mark Deliverable as Done** | `done INDEX` <br> e.g., `done 2`
+**Switch** | `switch MODE` <br> e.g., `switch deliverable`
```

Comment:
The commas in `e.g.,` seem to be a typo. I could be missing some markdown formatting.